From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Hans Wennborg <hans@chromium.org>
Date: Mon, 6 Mar 2023 08:35:40 +0000
Subject: Roll clang+rust llvmorg-17-init-3170-g6e30dffe-1 :
 llvmorg-17-init-3874-g93a2fecc-1 / [skipping Rust]

https://chromium.googlesource.com/external/github.com/llvm/llvm-project/+log/6e30dffe..93a2fecc

Ran: ./tools/clang/scripts/upload_revision.py 93a2feccbc909230b6a091f5836b92739bd12941

Bug: 1420777
Change-Id: I7866b5a59e34f7427f8b0eda32e7208806bd9056
Cq-Include-Trybots: chromium/try:chromeos-amd64-generic-cfi-thin-lto-rel
Cq-Include-Trybots: chromium/try:dawn-win10-x86-deps-rel
Cq-Include-Trybots: chromium/try:linux-chromeos-dbg
Cq-Include-Trybots: chromium/try:linux_chromium_cfi_rel_ng
Cq-Include-Trybots: chromium/try:linux_chromium_chromeos_msan_rel_ng
Cq-Include-Trybots: chromium/try:linux_chromium_msan_rel_ng
Cq-Include-Trybots: chromium/try:mac11-arm64-rel,mac_chromium_asan_rel_ng
Cq-Include-Trybots: chromium/try:ios-catalyst
Cq-Include-Trybots: chromium/try:win-asan
Cq-Include-Trybots: chromium/try:android-official,fuchsia-official
Cq-Include-Trybots: chromium/try:mac-official,linux-official
Cq-Include-Trybots: chromium/try:win-official,win32-official
Cq-Include-Trybots: chromium/try:linux-swangle-try-x64,win-swangle-try-x86
Cq-Include-Trybots: chrome/try:iphone-device,ipad-device
Cq-Include-Trybots: chrome/try:linux-chromeos-chrome
Cq-Include-Trybots: chrome/try:win-chrome,win64-chrome,linux-chrome,mac-chrome
Cq-Include-Trybots: chrome/try:linux-pgo,mac-pgo,win32-pgo,win64-pgo
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/4308214
Reviewed-by: Hans Wennborg <hans@chromium.org>
Commit-Queue: Hans Wennborg <hans@chromium.org>
Reviewed-by: Alan Zhao <ayzhao@google.com>
Cr-Commit-Position: refs/heads/main@{#1113280}
(cherry picked from commit 70c55df468405dd693f65d21bdfbe8db8b44b393)

diff --git a/tools/clang/scripts/update.py b/tools/clang/scripts/update.py
index 44dfe2f1f6df6397ec4c2684f682ab458f357f54..5b2289d07b9f23302a346aae180a8e18d39b9510 100755
--- a/tools/clang/scripts/update.py
+++ b/tools/clang/scripts/update.py
@@ -35,7 +35,7 @@ import zlib
 # https://chromium.googlesource.com/chromium/src/+/main/docs/updating_clang.md
 # Reverting problematic clang rolls is safe, though.
 # This is the output of `git describe` and is usable as a commit-ish.
-CLANG_REVISION = 'llvmorg-17-init-2387-g68e81d7e'
+CLANG_REVISION = 'llvmorg-17-init-3874-g93a2fecc'
 CLANG_SUB_REVISION = 1
 
 PACKAGE_VERSION = '%s-%s' % (CLANG_REVISION, CLANG_SUB_REVISION)
